src/definitions/modules/dropdown.js

```diff
@@ -200,6 +200,7 @@ $.fn.dropdown = function(parameters) {
           select: function() {
             if(module.has.input() && selectObserver) {
               selectObserver.observe($module[0], {
+                attributes: true,
                 childList : true,
                 subtree   : true
               });
```

Comment:
Note) See https://developer.mozilla.org/en-US/docs/Web/API/MutationObserverInit for options.